testing/munin/APKBUILD

```diff
@@ -0,0 +1,72 @@
+# Contributor: Stefan Wagner <stw@bit-strickerei.de>
+# Maintainer: Stefan Wagner <stw@bit-strickerei.de>
+pkgname=munin
+pkgver=2.0.25
+pkgrel=0
+pkgdesc="A distributed monitoring/graphing tool"
+url="http://munin-monitoring.org/"
+arch="noarch"
+license="GPL"
+perl_modules="perl-rrd perl-net-snmp perl-log-log4perl perl-html-template
+perl-net-ssleay perl-net-server perl-date-manip perl-io-socket-inet6
+perl-file-copy-recursive perl-fcgi perl-uri"
+depends="$pkgname-node bash rrdtool perl $perl_modules"
+depends_dev="perl perl-module-build"
+makedepends="$depends_dev"
```

Comment:
Since you don’t provide `-dev` subpackage, it doesn’t make sense to declare `depends_dev`. `makedepends` is for build/compile dependencies, `depends_dev` is for dependencies of `-dev` subpackage.